core/src/test/java/org/apache/accumulo/core/conf/SiteConfigurationTest.java

```diff
@@ -75,7 +77,9 @@ public void testFile() {
     assertEquals("mysecret", conf.get(Property.INSTANCE_SECRET));
     assertEquals("hdfs://localhost:8020/accumulo123", conf.get(Property.INSTANCE_VOLUMES));
     assertEquals("123s", conf.get(Property.GENERAL_RPC_TIMEOUT));
-    assertEquals("256M", conf.get(Property.TSERV_WALOG_MAX_SIZE));
+    @SuppressWarnings("deprecation")
+    Property deprecatedProp = Property.TSERV_WALOG_MAX_SIZE;
+    assertEquals("256M", conf.get(conf.resolve(Property.TSERV_WAL_MAX_SIZE, deprecatedProp)));
```

Comment:
Since this test is just testing that we can read the file correctly, we don't need to check the deprecated prop, just the new one, so there's no need to do `conf.resolve` here. I see that you already updated the config file to use the new prop, anyway, so just checking the new prop should suffice.

That is, unless you wanted this to serve as a test for `conf.resolve` itself... but I assume that's already unit tested elsewhere.